src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -80,6 +124,6 @@ public void safeState() {
         getData().motorSetPoint = 0;
 
         // Stops movement of air
-        getData().solenoidSetPoint = Value.kOff;
+        // getData().solenoidSetPoint = Value.kOff;
```

Comment:
Lets uncomment this since we aren't trying to use this to stop the intake spinning